vector/src/main/java/im/vector/app/features/home/room/detail/widget/RoomWidgetItem.kt

```diff
@@ -43,7 +43,7 @@ abstract class RoomWidgetItem : EpoxyModelWithHolder<RoomWidgetItem.Holder>() {
     override fun bind(holder: Holder) {
         super.bind(holder)
         holder.widgetName.text = widget.name
-        holder.widgetUrl.text = tryOrNull { URL(widget.computedUrl) }?.host ?: widget.computedUrl
+        holder.widgetUrl.text = tryOrNull { URL(widget.widgetContent.url) }?.host ?: widget.widgetContent.url
```

Comment:
Why are you using url instead of computed?

Reply:
We just need the `host` here. OK for you?